devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -0,0 +1,634 @@
+}
+
+private isFlushDimmer(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0051")
+}
+
+private isDINDimmer(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0052")
+}
+
+private isFlushDimmer010V(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0053")
+}
+
+private getParameterMap() {[
+	[
+		name: "Input 1 switch type", key: "input1SwitchType", type: "enum",
+		parameterNumber: 1, size: 1, defaultValue: 0,
+		values: [
+			0: "Default value - Mono-stable switch type (push button) – button quick press turns between previous set dimmer value and zero)",
+			1: "Bi-stable switch type ",
+			2: "Potentiometer (Flush Dimmer 0-10V is using set value the last received from potentiometer or from z-wave controller)",
+			3: "0-10V Temperature sensor (regulated output)"
+		],
+		description: "Set input based on device type (switch, potentiometer, temperature sensor,..)." +
+			"After parameter change to value 3 first exclude module (without setting parameters to default value) then wait at least 30s and then re include the module! "
+	],
+	[
+		name: "Input I1 Sensor reporting", key: "inputI1SensorReporting", type: "range",
+		parameterNumber: 140, size: 2, defaultValue: 5,
+		range: "1..10000",
+		description: "Input I1 Sensor reporting If analogue sensor is connected, module reports measured value on change defined by this parameter. " +
+			"0 Reporting disabled, " +
+			"5 (Default value) = 0,5 change, " +
+			"1 - 10000 = 0,1 - 1000 step is 0,1"
+	],
+	[
+		name: "Enable/Disable Double click function", key: "enable/DisableDoubleClickFunction", type: "boolean",
+		parameterNumber: 21, size: 1, defaultValue: 0,
+		optionInactive: 0, inactiveDescription: "Default value - Double click disabled",
+		optionActive: 1, activeDescription: "Double click enabled",
+		description: "If Double click function is enabled, a fast double click on the push button will set dimming power at maximum dimming value. " +
+			"Valid only if input is set as mono-stable (push button)."
+	],
+	[
+		name: "Saving the state of the device after a power failure", key: "savingTheStateOfTheDeviceAfterAPowerFailure", type: "boolean",
+		parameterNumber: 30, size: 1, defaultValue: 0,
+		optionInactive: 0, inactiveDescription: "Default value - dimmer module saves its state before power failure (it returns to the last position saved before a power failure)",
+		optionActive: 1, activeDescription: " Flush Dimmer 0-10V module does not save the state after a power failure, it returns to off position",
+		description: "Based on the parameter settings the stores/does not store the last value of the output after power failure. "
+	],
+	[
+		name: "Dimming time (soft on/off)", key: "dimmingTime(SoftOn/Off)", type: "range",
+		parameterNumber: 65, size: 2, defaultValue: 100,
+		range: "50..255",
+		description: "Set value means time of moving the dimmer between min. and max. dimming values by short press of push button or controlled through UI (BasicSet). " +
```

Comment:
"The time it takes for the dimmer to transition between min and max brightness after a short press of the button or when controlled through the UI"

Reply:
Fixed